index.js

```diff
@@ -288,6 +282,8 @@ export function execaSync(rawFile, rawArgs, rawOptions) {
 	};
 }
 
+const normalizeSyncOptions = (options = {}) => options.node && !options.ipc ? {...options, ipc: false} : options;
```

Comment:
`node: true` can be used with synchronous methods, but `ipc: true` cannot.
Therefore, `ipc: true` is not added when using `node: true` with synchronous methods.